yt/data_objects/selection_objects/cut_region.py

```diff
@@ -27,15 +27,15 @@ class YTCutRegion(YTSelectionContainer3D):
         A list of conditionals that will be evaluated.  In the namespace
         available, these conditionals will have access to 'obj' which is a data
         object of unknown shape, and they must generate a boolean array.  For
-        instance, conditionals = ["obj['temperature'] < 1e3"]
+        instance, conditionals = ["obj['gas', 'temperature'] < 1e3"]
```

Comment:
Oh I figured out why! Anyways, this doesn't need to be updated in this PR but I think it would be good later on to make these tuples so we're consistent in the codebase.

Reply:
I _tried_ to use consistently the latter format, but I obviously missed a few of them. I think the ideal outcome would be to have that enforced by `black` :)

Reply:
I don't think black can do that, for reference, because in the general case, a two-element key isn't equivalent to a single two-size tuple one.

Reply:
Agreed!